OutOfSchool/OutOfSchool.WebApi/Services/ApplicationService.cs

```diff
@@ -492,11 +493,30 @@ public async Task<int> ChangeApprovedStatusesToStudying()
         return result;
     }
 
-    private static void UpdateStatus(ApplicationUpdate applicationDto, Application application, bool isUserProvider)
+    private static void UpdateStatus(ApplicationUpdate applicationDto, Application application, ICurrentUserService currentUserService)
```

Comment:
`currentUserService` is a class field, so maybe having this method as static is unnecessary? what do we get for passing the class field as argument vs direct access as instance member? :)

Reply:
Fix it )))